br-ext/package/optee_rust_examples_ext/optee_rust_examples_ext.mk

```diff
@@ -28,7 +29,11 @@ export CC = $(OPTEE_DIR)/toolchains/aarch64/bin/aarch64-linux-gnu-gcc
 define OPTEE_RUST_EXAMPLES_EXT_BUILD_CMDS
 	@$(foreach f,$(wildcard $(@D)/examples/*/Makefile), \
 		echo Building $f && \
-		$(OPTEE_RUST_EXAMPLES_TC_PATH_ENV) $(MAKE) -C $(dir $f) &&) true
+		$(OPTEE_RUST_EXAMPLES_TC_PATH_ENV) \
+		CROSS_COMPILE="$(shell echo $(BR2_PACKAGE_OPTEE_RUST_EXAMPLES_EXT_CROSS_COMPILE))" \
```

Comment:
Why is the `$(shell echo ...)` part needed?

Reply:
It was copied from `optee_examples_ext.mk` but it works without them too. So I will drop `$(shell echo ...)` part.